Lekce_02_Ukol02/Program.cs

```diff
@@ -4,20 +4,30 @@ internal class Program
     {
         static void Main(string[] args)
         {
+            Console.WriteLine("Zadej počet hvězdiček: ");
+            string vstup = Console.ReadLine();
+
             try
             {
-                Console.WriteLine("Zadej počet hvězdiček: ");
-                string vstup = Console.ReadLine();
                 int pocetHvezdicek = int.Parse(vstup);
 
+                if (pocetHvezdicek < 0)
+                {
+                    throw new ArgumentOutOfRangeException("Počet hvězdiček nesmí být záporný");
+                }
+
                 for (int i = 0; i < pocetHvezdicek; i++)
                 {
                     Console.Write("*");
                 }
             }
-            catch
+            catch (FormatException exception)
+            {
+                Console.WriteLine($"Došlo k výjimce: {exception.Message}");
+            }
+            catch (ArgumentOutOfRangeException exception)
             {
-                Console.WriteLine("Zadal jsi neplatný vstup.");
+                Console.WriteLine($"Došlo k výjimce: {exception.Message}");
             }
```

Comment:
Tohle se da napsat jednoduseji, protoze v obou pripadech to handlujes stejne. To se da udelat skrze `when` exception filtr :) 
```csharp
catch (Exception exception) when (exception is FormatException or ArgumentOutOfRangeException)
{
    Console.WriteLine($"Došlo k výjimce: {exception.Message}");
}
```

Reply:
opraveno